examples/master/README.md

```diff
@@ -89,20 +98,30 @@ video that shows the full workflow. What happens in the video is:
    When this job finishes, the Pod is automatically destroyed by the Kubernetes
    plugin.
 
-2. If the unit tests passed, the `sample-app-build` is started automatically via
+2. If the unit tests in `sample-app-test` passed and the job completes successfully, the `sample-app-build` job is started automatically via
    the Jenkins [promoted builds](https://wiki.jenkins-ci.org/display/JENKINS/Promoted+Builds+Plugin)
-   plugin. This job will leverage the OpenShift Jenkins plugin and start
-   build of the Docker image which will contain 'sample-app'.
+   plugin. This job will leverage the [OpenShift Jenkins Pipeline plugin](https://github.com/openshift/jenkins-plugin) and start an OpenShift
+   build that creates a Docker image that contains 'sample-app'.  The build results will be validated, as well as whether a deployment
+   and replication controller based on the resulting image are available (with a replica count of 0).
 
 3. Once the new Docker image for the `sample-app` is built, the
-   `sample-app-stage` project will automatically deploy it into `stage` project
-   and notify the QA team about availability for testing.
-
-3. If the `sample-app` image passes the `stage` testing, you have to **manually
-   promote** the `sample-app-build` build to be deployed to OpenShift. Since
-   re-deploying the application replaces the existing running application, human
-   intervention is needed to confirm this step.
-
-4. Once the build is promoted, the `sample-app-deploy` job is started. This job
-   will scale down the existing application deployment and redeploy it using the
-   new Docker image.
+   `sample-app-stage` job will validate that the new Docker imageis successfully deploy with a running replic into the `stage` project.  The
+   deployment validation is started automatically because the `sample-app-stage` job continually monitors the ImageStream where the image resides
+   in the `ci` project and will detect any change in the ImageStream produced by the `sample-app-build` job.
+
+   A sample polling interval has been provided with the sample, but alter the interval to best suit your needs.  Also, no email notification is
+   preconfigured (a running SMTP server and a valid email are needed for the job to succeed), but you can set up SMTP in your Jenkins environment and
+   add a Jenkins email post build action to notify the QA team about your new image's availability for testing.
+   
+   Also note, the `sample-app-stage` job will start polling the `ci` project as soon as the Jenkins master comes up.  Jenkins' SCM polling infrastructure will 
+   trigger the job at least once because there are no existing job results, irregardless of the SCM commit ID polling information returned.  So you'll see
```

Comment:
is this a jenkins limitation or our limitation?  doesn't sound like a great user experience.